packages/bvaughn-architecture-demo/components/Initializer.tsx

```diff
@@ -3,7 +3,8 @@
 
 import { ReactNode, useContext, useEffect, useRef, useState } from "react";
 
-import { ReplayClientContext } from "../src/contexts/ReplayClientContext";
+import { ReplayClientContext } from "../../shared/client/ReplayClientContext";
```

Comment:
:speech_balloon: fyi, declaring `shared` as a dep and installing it would let us drop the `"../../"` prefix, although that may work better once we switch package managers

Reply:
I had trouble getting this to work. I'll try again!